src/marshaller/HipieDDL.js

```diff
@@ -314,8 +330,57 @@
                     retVal.push(visualization);
                 }
             }, this);
-        } else if (exists("_updates.visualization", this)) { //TODO For backward compatability - Remove in the future  ---
-            retVal.push(this.visualization.dashboard.visualizations[this._updates.visualization]);
+        }
+        return retVal;
+    };
+
+    function Events(visualization, events) {
+        this.visualization = visualization;
+        this.events = {};
+        for (var key in events) {
+            this.events[key] = new Event(visualization, key, events[key]);
+        }
+    };
+
```

Comment:
How does this events object work, exactly is this something we will interact with?